infrastructure/k8s/bin/helmit.py

```
@@ -75,13 +80,28 @@ def render_cronjob(project_name, secret_keys, deployment):
     context.update(deployment)
     format_values(context)
     template = yaml.load(open("templates/cron-job.yaml"), Loader=yaml.SafeLoader)
-    print("---")
-    print(yaml.dump(jsone.render(template, context)))
-
+    # print("---")
+    # print(yaml.dump(jsone.render(template, context)))
+    write_file(template, context, "cron")
+
+def write_file(template, context, prefix):
+    filename = chartsdir + "/taskcluster-" + prefix + "-" + p.split("/")[-1:][0]
+    try:
+        f = open(filename, "a+")
+        f.write(yaml.dump(jsone.render(template, context)))
+        f.close()
+    except: 
+        print("failed to write "+ filename)
 
 parser = argparse.ArgumentParser()
 parser.add_argument("--service", help="Name of the service to render", default=None)
 args = parser.parse_args()
+chartsdir = "charts"
+
+try:
+    os.mkdir(chartsdir)
+except:
```

Only `pass` on `FileExistsError`. Otherwise we risk ignoring errors like insufficient permissions to create the dir.